config/src/sumeragi.rs

```diff
@@ -53,7 +54,7 @@ view! {
     }
 }
 
-impl Default for Configuration {
+impl DocsDefault for Configuration {
```

Comment:
Not a huge fan of `DocsDefault` for a trait name. Maybe `ReferenceDefault`? `ExampleDefault`?

Reply:
Also, the point is that we shouldn't allow people anything resembling a `Default` trait. The point is to prevent this kind of usage, which we can achieve by 
1. enforcing that the default is either implemented only for the proxy structure, and no concrete `Configuration` `impl Default` which we can check. 
2. By encoding the default values in place where they're used. So there isn't a general `Default` trait, just an example struct instance.